### PR TITLE
Expand the release instructions and put them inside the maintainer's guide

### PR DESCRIPTION
**Pull request by eternoendless**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Improve the release guide with more detail and break it into sections.
| Fixed ticket? | ~

### CODE REVIEW

File: src/content/1.7/project/maintainers-guide/releasing-prestashop/create-build.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+- the Changelog file – including a list of all the merged Pull Requests. Make sure to keep this file, you'll need it later.
+- the New Contributors file – a list of all the people who contributed code for this version for their first time.
+
+### Update the files in the project
+
+- Add the contents of the changelog at the top end of PrestaShop's [Changelog file][changelog-file].
+- Update PrestaShop's [Contributors file][contributors-file] by adding the new contributors.
+- Commit your changed files with following message: `// Changelog [version]`
+- You may push this branch to your own fork, if you wish to.
```
Comment (atomiix) [inline src/content/1.7/project/maintainers-guide/releasing-prestashop/create-build.md:64]: Maybe on a **private** fork if it contains security fixes?

File: src/content/1.7/project/maintainers-guide/releasing-prestashop/create-build.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+    ```shell
+    git clone git@github.com:PrestaShop/PrestaShop.git
+    ```
+
+* **Make sure that you switch to the appropriate branch** regarding the version you'll be building (e.g. you must be on branch 1.7.7.x if you're building a 1.7.7 release).
+
+* **Make sure your branch is up-to-date with upstream.** Especially if you already had a local clone of the repository.
+
+* **Create a local branch for you work.** Keep it! You will need to go back to it later.
```
Comment (matks) [inline src/content/1.7/project/maintainers-guide/releasing-prestashop/create-build.md:28]:
```suggestion
* **Create a local branch for your work.** Keep it! You will need to go back to it later. 
```

File: src/content/1.7/project/maintainers-guide/releasing-prestashop/create-build.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+By default, the release package will create two files in a new subdirectory in `/tools/build/releases`:
+
+- A ZIP package, containing the software.
+- An XML file, containing md5 checksums for every file within the package.
+
+{{% notice warning %}}
+**Make sure to keep them both!** You will need them later on.
+{{% /notice %}}
+
+As an optional step, consider downloading the latest stable release package and compare the contents of the zip archieves to look for suspicious changes.
```
Comment (matks) [inline src/content/1.7/project/maintainers-guide/releasing-prestashop/create-build.md:99]:
```suggestion
As an optional step, consider downloading the latest stable release package and compare the contents of the zip archives to look for suspicious changes.
```

File: src/content/1.7/project/maintainers-guide/releasing-prestashop/create-build.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+
+For patch versions, the beta and RC phase can be skipped (example: `prestashop_1.7.4.1+build.1.zip`)
+{{% /callout %}}
+
+### Upload files to the archive
+
+{{% notice warning %}}
+**This step requires special rights.**
+
+Send both files to a maintainer from the PrestaShop Company with access to the Archive Drive to perform this step.
```
Comment (matks) [inline src/content/1.7/project/maintainers-guide/releasing-prestashop/create-build.md:133]:
```suggestion
Send ZIP and XML files to a maintainer from the PrestaShop Company with access to the Archive Drive to perform this step.
```

File: src/content/1.7/project/maintainers-guide/releasing-prestashop/final-steps.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+* Check the new release can be downloaded from the prestashop.com website and GitHub:
+
+    - [GitHub releases](https://github.com/PrestaShop/PrestaShop/releases)
+    - [PrestaShop.com archives](https://www.prestashop.com/en/previous-versions)
+
+* Check that if you try to install PrestaShop 1.7.5 from the archive, the installer suggests you install the latest version instead
+* Check that the release note has been published on the [Build Blog](http://build.prestashop.com)
+* Check that the [PrestaShop.com](https://www.prestashop.com) shows the right "latest version", and the links are correct
+* Check that the latest release has an available docker image on the [Docker repository][docker-repository]
+* Check that the [public demo](http://demo.prestashop.com) runs on the latest version (a few hours after the docker image release)
```
Comment (matks) [inline src/content/1.7/project/maintainers-guide/releasing-prestashop/final-steps.md:67]:
```suggestion
* Check that the [public demo](https://demo.prestashop.com) runs on the latest version (a few hours after the docker image release)
```

File: src/content/1.7/project/maintainers-guide/releasing-prestashop/release-publicly.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+  - If you're publishing a patch version:
+    - Write a short intro explaining what the release fixes – especially if it's a security release! ([Example](https://github.com/PrestaShop/PrestaShop/releases/tag/1.7.5.1))
+
+- Upload the new version zip archive (the one that has been validated by QA).  
+  Remove the build number from the file name to avoid confusions.
+
+{{% callout %}}
+##### If the Changelog content is too long
+
+It's not inusual to have very long changelogs, especially in minor versions. To reduce its size, we recommend putting it in a collapsable paragraph:
```
Comment (matks) [inline src/content/1.7/project/maintainers-guide/releasing-prestashop/release-publicly.md:71]:
```suggestion
It's not unusual to have very long changelogs, especially in minor versions. To reduce its size, we recommend putting it in a collapsable paragraph:
```

**Review by matks — CHANGES_REQUESTED** [review]
Content is OK for me 👍  . Only some typos to fix, and @atomiix feedback

File: src/content/1.7/project/maintainers-guide/releasing-prestashop/create-build.md (hunk 1 above)
Comment (eternoendless) [inline src/content/1.7/project/maintainers-guide/releasing-prestashop/create-build.md:64]:
```suggestion
- You may push this branch to your own (private) fork, if you wish to.
```

Comment (eternoendless) [reply src/content/1.7/project/maintainers-guide/releasing-prestashop/create-build.md:64]:
```suggestion
- You may push this branch to your own (private) fork, if you wish to.
```

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @eternoendless

**Pull request merged**